scripts/deploy-backend.sh

```diff
@@ -104,6 +104,9 @@ update_backend()
 
     cf map-route "$CGAPPNAME_BACKEND" apps.internal --hostname "$CGAPPNAME_BACKEND"
 
+    # Add network policy to allow frontend to access backend
+    cf add-network-policy "$CGAPPNAME_FRONTEND" "$CGAPPNAME_BACKEND" --protocol tcp --port 8080
+
```

Comment:
In your how-to-test section, you state this: `make sure the network policy is deleted before deployment`

Could we devise a way to make the script here more robust and not need such a precondition? 

cc @George-Hudson

Reply:
The precondition is only for testing, so that we can see it is created. I tested this with existing policy and it did not complain. So if it doesn't exist, then it will create it, otherwise nothing happens